src/main/java/com/google/graphgeckos/dashboard/storage/DatastoreRepository.java

```diff
@@ -171,17 +178,24 @@ public boolean deleteRevisionEntry(@NonNull String commitHash) {
   /**
    * {@inheritDoc}
    *
-   * @throws NullPointerException if the {@code commitHash} is null.
+   * @throws NullPointerException if the {@code commitHash} is null
    */
   @Override
   public BuildInfo getRevisionEntry(@NonNull String commitHash) {
-    Objects.requireNonNull(commitHash);
+    checkNotNull(commitHash, "commitHash should not be null");
```

Comment:
I think the message is unnecessary

Reply:
I'll remove them before merging. Thank you!